### DISCUSSION TITLE
Filtering low count features in scATAC data

### DISCUSSION THREAD

**Discussion by wudustan** [stuart-lab/signac#833] [Q&A] [answered]
I have an scATAC dataset of 5487 cells x 218666 features. I have obtained rowSums of the feature counts and have plotted the result sorted from lowest to highest value.
<img width="910" alt="Screenshot 2021-10-19 at 10 41 43" src="https://user-images.githubusercontent.com/15104192/137885639-aca0310f-4c93-4d99-881c-66b7eb79529d.png">

As you can see, around 20,000 features have total counts below 100. My question is, can (?should) I simply remove these features from my data after normalisation?

**Comment by timoast** [comment, marked as answer]
You can remove them if you think they're more likely to be some technical artefact rather than capturing biological variation among the cells. In my experience, retaining peaks like this does not have much of an impact on downstream analysis results like clustering.

**Comment by wudustan** [comment]
Thank you
